src/back/spv/mod.rs

```diff
@@ -39,7 +42,7 @@ pub(crate) struct LogicalLayout {
     function_definitions: Vec<Word>,
 }
 
-struct Instruction {
+pub(crate) struct Instruction {
```

Comment:
Yes that is true, I changed it to `pub(self)`, I have to mark it as a `pub` function, otherwise I am leaking the visibility in the function `validate_spec_requirements`. Anyway, `pub(self)` is far better than using `pub(crate)` 👍